FreydCategoriesForCAP/gap/FreydCategory.gi

```diff
@@ -1745,6 +1745,223 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_FREYD_CATEGORY,
       
     fi;
     
+    ## Random Methods by Lists
+    ##
+    if ForAll( [ "RandomObjectByList",
+                 "RandomMorphismWithFixedSourceByList",
+                 "RandomMorphismWithFixedRangeByList",
+                 "RandomMorphismByList",
+                 "WeakBiPushout",
```

Comment:
Please check for the operations actually used (`InjectionOfFirstCofactorOfWeakBiPushout`, `InjectionOfSecondCofactorOfWeakBiPushout`, `ProjectionInSecondFactorOfWeakBiFiberProduct`).